jellyfin_kodi/objects/kodi/queries.py

```diff
@@ -409,12 +409,12 @@
 add_set_obj = ["{Title}", "{Overview}"]
 add_video_version = """
 INSERT INTO     videoversion(idFile, idMedia, media_type, itemType, idType)
-VALUES          (?, ?, ?, ?, ?)
+VALUES          (?, ?, ?, (SELECT itemType FROM videoversiontype WHERE id = ?), ?)
```

Comment:
What is the performance impact of this additional inner SELECT?

@coderabbitai maybe you can help with this question?
These are sqlite database queries.